docs/source/user-guide/vision/supported_tasks_and_formats.rst

```diff
@@ -0,0 +1,124 @@
+for each supported task type.
+
+The data formats are validated at the creation of the VisionData object based on the first batch of data. However,
+in addition to the automatic validations, it is also important to make sure visually that the data was provided in the
+correct format via the :func:`head <deepchecks.vision.VisionData.head>` functionality.
+
+
+Common Formats
+==============
+All types of data are loaded in a batch by batch manner meaning that each data input contain
+information about several samples. In order to support this, all data inputs are expected to be provided
+as either an iterable in which each entry is the data for a single sample or as a high dimension array or tensor in
+which the first dimension is the number of samples.
+
+Image Format
+------------
+Each image in the batch can either be a `PIL image object <https://pillow.readthedocs.io/en/stable/reference/Image.html>`_
+or a ``[H, W, C]`` 3D array (can be either numpy or a tensor).
+For the array format, the first dimension must be the image y axis, the second being the image x axis, and the
+third being the number of channels.
+The numbers in the array should be integers in the range [0, 255]. Color images should be in RGB format and
+have 3 channels, while grayscale images should have 1 channel.
+
+
+Image Identifiers Format
+------------------------
+Image identifiers is a mechanism that allow identifying relevant images in the check results so that they can
+be easily retrieved from the original dataset by the user. Common uses for this can be identifying images by
+the path to the image or by their name or id in a database.
+Image identifiers need to be provided as an iterable of strings.
+
+.. _supported_tasks__classification:
+Classification
+==============
+Label Format
+------------
+Classification label per sample is expected to be provided as either a string or an int representing the class index.
+
+Prediction Format
+-----------------
+Classification prediction per sample should be **prediction probabilities** per class, meaning that it should be an
+iterable of floats in the range [0, 1] with length equal to the number of classes.
+
+.. _supported_tasks__object_detection:
+Object Detection
+================
+Label Format
+------------
+Object detection label per sample should be an iterable of bounding boxes. Each bounding box should be an iterable
+of 5 elements in the following order: ``(class_id, x_min, y_min, w, h)``.
+
+    ``x_min`` and ``y_min`` are the coordinates (in pixels) of the **top left corner** of the bounding box, ``w``
+    and ``h`` are the width and height of the bounding box (in pixels) and ``class_id`` is the class id of the label.
+
+For example, labels of a batch containing 2 samples, where the first sample has a single bounding box and the second
+sample has 2 bounding boxes, should be provided as follows:
+
+.. code-block:: python
+
+    [
+        [(1, 50, 20, 9.4, 5.5)],
```

Comment:
Use ints (round numbers) for coordinates.

Reply:
the floats are for hight and weight

Reply:
still should be ints